dev_tools/notebooks/isolated_notebook_test.py

```diff
@@ -90,10 +95,22 @@ def _find_base_revision():
     raise ValueError("Can't find a base revision to compare the files with.")
 
 
+def _list_all_notebooks() -> Set[str]:
+    try:
+        output = subprocess.check_output(['git', 'ls-files', '*.ipynb'])
+        return set(output.decode('utf-8').splitlines())
+    except subprocess.CalledProcessError as ex:
+        warning("It seems that tests are run from not a git repo, notebook tests are skipped", ex)
```

Comment:
nit: misplaced "not", also s/run/running. Suggestion: "It seems that tests are not running in a git repo, skipping notebook tests"

Reply:
OK, as this was duplicated across the two test files, I decided to factor them out into a `utils.py` instead of modifying this sentence in both places. It grew the scope but it was the right thing to do. Can you PTAL again? Thanks!

Reply:
I'm going to go ahead and merge this :) thanks for the review. Please let me know if you find any other nits!